# 标签名称
class_name: ['plastic','plastic bottle','can']
robot:
  base_frame_id: base_link_rm
  ee_frame_id: ee_link
camera:
  frame_id: camera_color_optical_frame
  color_topic: /camera/color/image_raw
  info_topic: /camera/aligned_depth_to_color/camera_info
  depth_topic: /camera/aligned_depth_to_color/image_raw
# 阈值设置
threshold:
  iou: 0.4
  confidence: 0.7
  # 颜色与面积阈值
  color_threshold:
    roi_min: [0, 0, 100]
    roi_max: [60, 60, 180]
    lower: [100, 25, 100]
    upper: [255, 255, 210]
  area_threshold:
    min_area: 1300
    max_area: 6000

# 计算设备
# -cpu
# -openvino
# -ort
# -trt
device: 'trt'
model_type: 'yolov5'
# 模型权重路径
model_file: /home/ydfo/rwm_moveit/src/robot_control/robot_camera_control/model/ribbish.onnx
params_file: ""
config_file: ""

#grasp_net
gmodel_file: "/home/ydfo/rwm_moveit/src/robot_control/robot_camera_control/model/checkpoint-rs.tar"
num_point: 20000
num_view: 300
collision_thresh: 0.01
voxel_size: 0.01